src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/Validation/collection-objects-naming.json

```diff
@@ -0,0 +1,150 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "Operations schemes has non-http/https type",
+    "description": "Some documentation.",
+    "version": "2014-04-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/",
+  "paths": {
+    "/foo": {
+      "post": {
+        "operationId": "Models_list",
+        "summary": "Foo path",
+        "description": "Foo operation",
+        "responses": {
+          "default": {
+            "description": "Unexpected error"
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Models"
+            }
+          }
+        },
+        "produces": [
```

Comment:
minor: I'd push all the produces and consumes to top level as we have one SHOULD rule that'll throw warning otherwise :)

In general, I'd try to avoid all other kind of errors/warning except the one i am testing this for.